Game/Minigames/ClickAtTheRightTime/ClickAtTheRightTime.gd

```diff
@@ -136,7 +133,7 @@ func _process(delta):
 			flatStyle.bg_color = Color.black
 	
 func getCursorPosition():
-	return (sin(time*cursorSpeed) + 1.0)/2.0
+	return (sin(pow(time * cursorSpeed, 1.2)) + 1.0) / 2.0
```

Comment:
This makes it so the minigame gets harder and harder over time. Maybe keep it behind the hard setting?